crates/gui/src/components/TopBar.tsx

```diff
@@ -6,14 +6,17 @@ import { AddIcon, SyncIcon } from '../icon/icon'
 import Panel from './Panel'
 
 const TopBar = () => {
+    const [refresh, setRefresh] = createSignal(false)
     const [rid, setRid] = createSignal('')
     const [onInput, setInput] = createSignal(false)
     const [onPanel, setPanel] = createSignal(false)
     const [live, setLive] = createSignal('bili')
     return (
         <div data-tauri-drag-region class="top-bar">
             <button class="top-bar-btn">
-                <SyncIcon size={18} />
+                <div class={`refreshdiv ${refresh() ? 'refresh' : ''}`} onClick={() => {
+                    setRefresh(!refresh())
+                }}><SyncIcon size={18} /></div>
```

Comment:
```suggestion
                <div class={`refreshdiv ${refresh() ? 'refresh' : ''}`} onClick={() => {
                    setRefresh(!refresh())
                }}><SyncIcon size={18} /></div>
```

考虑使用 classList 的形式来写